### ISSUE TITLE
Stack Exchange login error

### ISSUE THREAD

**Issue by Six-wars** [pennersr/django-allauth#1639]
I get this error after trying to login with stack exchange 

```
Environment:


Request Method: GET
Request URL: http://kazichimp.com/accounts/stackexchange/login/callback/?code=CBy0SGksBsBjYZojGIsTwg))&state=jEssFyErtLnt

Django Version: 1.8.7
Python Version: 2.7.12
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'sorl.thumbnail',
 'django.contrib.sites',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'allauth.socialaccount.providers.facebook',
 'allauth.socialaccount.providers.google',
 'allauth.socialaccount.providers.linkedin_oauth2',
 'allauth.socialaccount.providers.twitter',
 'allauth.socialaccount.providers.stackexchange',
 'django.contrib.admin',
 'django_countries',
 'accounts',
 'common')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware')


Traceback:
File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/local/lib/python2.7/dist-packages/allauth/socialaccount/providers/oauth2/views.py" in view
  69.                 return self.dispatch(request, *args, **kwargs)
File "/usr/local/lib/python2.7/dist-packages/allauth/socialaccount/providers/oauth2/views.py" in dispatch
  130.                                                 response=access_token)
File "/usr/local/lib/python2.7/dist-packages/allauth/socialaccount/providers/stackexchange/views.py" in complete_login
  23.         extra_data = resp.json()['items'][0]

Exception Type: KeyError at /accounts/stackexchange/login/callback/
Exception Value: 'items'
```

**Comment by mauler** [comment]
@Six-wars  how did you fixed this issue ?

**Comment by pennersr** [comment]
Closed via 2aa94e9c

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/1639
